<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnatomyShowClass</class>
 <widget class="QMainWindow" name="AnatomyShowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>204</red>
        <green>232</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>204</red>
        <green>232</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Adobe 仿宋 Std R</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AnatomyShow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>20</width>
      <height>680</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtWizard">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>560</y>
      <width>781</width>
      <height>120</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>227</red>
          <green>227</green>
          <blue>227</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>227</red>
          <green>227</green>
          <blue>227</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Adobe 仿宋 Std R</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImageSegmentation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>160</width>
      <height>70</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>140</green>
          <blue>140</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>140</green>
          <blue>140</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>140</green>
          <blue>140</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图像分割</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMeshProcess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>205</y>
      <width>160</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mesh处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnatomyDefine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>372</y>
      <width>160</width>
      <height>68</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>140</green>
          <blue>140</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>140</green>
          <blue>140</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>140</green>
          <blue>140</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>anatomy定义</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnatomyShow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>508</y>
      <width>160</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>anatomy 显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblShowFirstStep">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblShowSecondStep">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>70</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblShowThirdStep">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>70</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actProcessImageSegmentation"/>
   <addaction name="actProcessFromMesh"/>
   <addaction name="actFromAnatomyDefine"/>
  </widget>
  <action name="actProcessImageSegmentation">
   <property name="text">
    <string>从图像分割开始</string>
   </property>
  </action>
  <action name="actProcessFromMesh">
   <property name="text">
    <string>从mesh开始</string>
   </property>
  </action>
  <action name="actFromAnatomyDefine">
   <property name="text">
    <string>从人体结构定义开始</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="anatomyshow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actProcessFromMesh</sender>
   <signal>triggered()</signal>
   <receiver>AnatomyShowClass</receiver>
   <slot>ToolbarProcessFromMeshClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actFromAnatomyDefine</sender>
   <signal>triggered()</signal>
   <receiver>AnatomyShowClass</receiver>
   <slot>ToolbarProcessFromAnatomyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actProcessImageSegmentation</sender>
   <signal>triggered()</signal>
   <receiver>AnatomyShowClass</receiver>
   <slot>ToolbarProcessFromImageClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMeshProcess</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyShowClass</receiver>
   <slot>MeshProcessClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImageSegmentation</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyShowClass</receiver>
   <slot>ImageSegmentationClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAnatomyDefine</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyShowClass</receiver>
   <slot>AnatomyDefineClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAnatomyShow</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyShowClass</receiver>
   <slot>AnatomyShowClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ToolbarProcessFromImageClicked()</slot>
  <slot>ToolbarProcessFromMeshClicked()</slot>
  <slot>ToolbarProcessFromAnatomyClicked()</slot>
  <slot>ImageSegmentationClicked()</slot>
  <slot>MeshProcessClicked()</slot>
  <slot>AnatomyDefineClicked()</slot>
  <slot>AnatomyShowClicked()</slot>
  <slot>PrevStepFromMeshProcess()</slot>
 </slots>
</ui>
